docs/apache-airflow-providers-amazon/executors/general.rst

```diff
@@ -74,6 +74,13 @@ Then you can build your image by ``cd``-ing to the directory with the Dockerfile
    docker build -t my-airflow-image \
     --build-arg aws_default_region=YOUR_DEFAULT_REGION .
 
+For users on Apple Silicon, you must specify the arch using ``docker buildx``:
+
+.. code-block:: bash
+
+  docker buildx build --platform=linux/amd64 -t my-airflow-image \
```

Comment:
(driveby comment) It might be worth adding the platform in the dockerfile instead. Less likely to get forgotten that way.

```
FROM --platform=linux/amd64 someimage
```

Reply:
I'm wondering whether hardcoding the platform in the Dockerfile would cause problems or confusion for the user, if they didn't mean to build _and_  run under x86. 

I think it would be a better idea to leave this up to the user, should they decide they need to specify x86 architeture. What do you think?

Reply:
Any thoughts @jedcunningham ? :)

Reply:
Sorry, I don't have enough context to really have an opinion. My only point was that, if the platform is consistent, doing it in the dockerfile is less error prone than an extra arg in `docker build`. If the platform does vary, the extra arg is the way to go. Let's just say I have first hand knowledge that it gets missed sometimes :)